### ISSUE TITLE
How/where should we document ?locale= and ?lang= params

### ISSUE THREAD

**Issue by buren** [justarrived/just-match-web#480]
#450 adds support for `locale=` and `lang=` parameters, however they aren't that discoverable ;)

**Comment by bex1** [comment]
Yeah. Looking through the source and one may stumble upon them isn't an ideal scenario hehe. Maybe each page can have some documentation or each route in ja-routes if we want to keep it close to the code.

Or we add some seperate document for it.

**Comment by buren** [comment]
I'd like to have this kind of functionality discoverable(-ish) by non-devs, perhaps we could add some markdown file to a `docs/` folder or somehing like that? Where we can keep track of these sort of things.
